src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -236,12 +236,21 @@ private IList<string> GetResourceNamesForCulture(CultureInfo culture)
             {
                 var names = new List<string>();
```

Comment:
Oh yes, I had already made the change, was just noting that that wasn't the only change there.